src/serverOperations/readersServerOperations.js

```diff
@@ -0,0 +1,72 @@
+import { bases } from '../enums';
+import {
+    createRecord,
+    deleteRecord,
+    getRecords,
+    updateRecord,
+} from './crudOperations';
+import { fetchFromBase } from './fetchFromBase';
+
+export function getReadersServerOperations(sessionID) {
+    return {
+        createReader: async (name, surname, phoneNumber, email = '') => {
+            email = email ? email : undefined;
```

Comment:
This one line is absolutely ugly :D Please change it (see `||` and `??` operators)